packages/rrweb/src/record/mutation.ts

```diff
@@ -543,9 +543,11 @@ export default class MutationBuffer {
           }
         } else {
           // overwrite attribute if the mutations was triggered in same time
+          const element = m.target as HTMLElement;
           item.attributes[m.attributeName!] = transformAttribute(
             this.doc,
-            (m.target as HTMLElement).tagName,
+            element,
+            element.tagName,
```

Comment:
l: We probably don't need both of these parameters

Reply:
yeah, I thought the same, wasn't sure - in one call site there is this `getValidTagName` method, not sure if we need this here...?

Reply:
We can always revisit this/clean this up later.